docs/template_plugin/tests/functional/op_reference/prelu.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ie_core.hpp>
+#include <ie_ngraph_utils.hpp>
+#include <limits>
+#include <algorithm>
+#include <ngraph/ngraph.hpp>
+#include <shared_test_classes/base/layer_test_utils.hpp>
+
+#include "base_reference_test.hpp"
+
+using namespace reference_tests;
+using namespace ngraph;
+using namespace InferenceEngine;
+
+namespace {
+struct PreluParams {
+    template <class IT>
+    PreluParams(const PartialShape& shape, const element::Type& iType, const std::vector<IT>& iValues, const std::vector<IT>& oValues,
+                const Shape& slopeShape, const std::vector<IT>& negativeSlopeValues)
+        : pshape(shape),
+          inType(iType),
+          outType(iType),
+          inputData(CreateBlob(iType, iValues)),
+          refData(CreateBlob(iType, oValues)),
+          negativeSlopeShape(slopeShape),
+          negativeSlope(CreateBlob(iType, negativeSlopeValues)) {}
+
+    PartialShape pshape;
+    element::Type inType;
+    element::Type outType;
+    Blob::Ptr inputData;
+    Blob::Ptr refData;
+    Shape negativeSlopeShape;
+    Blob::Ptr negativeSlope;
+};
+
+class ReferencePreluLayerTest : public testing::TestWithParam<PreluParams>, public CommonReferenceTest {
+public:
+    void SetUp() override {
+        auto params = GetParam();
+        function = CreateFunction(params);
+        inputData = {params.inputData, params.negativeSlope};
+        refOutData = {params.refData};
+    }
+    static std::string getTestCaseName(const testing::TestParamInfo<PreluParams>& obj) {
+        auto param = obj.param;
+        std::ostringstream result;
+        result << "shape=" << param.pshape << "_";
+        result << "iType=" << param.inType << "_";
+        result << "oType=" << param.outType << "_";
+        result << "slopeShape=" << param.negativeSlopeShape;
+        return result.str();
+    }
+
+private:
+    static std::shared_ptr<Function> CreateFunction(const PreluParams& params) {
+        const auto in = std::make_shared<op::Parameter>(params.inType, params.pshape);
+        //const auto SLOPE = op::Constant::create(input_type, negativeSlopeShape, negativeSlope->get());
+        const auto SLOPE = std::make_shared<op::Parameter>(params.inType, params.negativeSlopeShape);
```

Comment:
@ilyachur @iefode PRelu ngraph backend test runs slope as both parameter and constant input. I can implement Parameter input as the code. But I cannot for Constant input.
-Can I extract pointer of vector from Blob::Ptr?
-If not, I can make std::vector<float>  only for fp32 as the ngraph backend test of prelu uses only fp32.